src/components/ExpensesList.js

```diff
@@ -5,22 +5,22 @@ const ExpensesList = ({ expenses, removeExpense, filteredExpenses }) => {
     return filteredExpenses.map((expense) => (
       <Expense key={expense.id} {...expense} removeExpense={removeExpense} />
     ));
-  };
+  }
 
   return (
     <section className="my-4">
       <div className="text-start">
         <h2>History</h2>
       </div>
       <ul className="list-group">
-        {expenses.length > 0 ? (
+        {expenses.length > 0 && filteredExpenses.length > 0 ? (
```

Comment:
added conditional to render also based on filteredExpenses

Reply:
I think since we only ever actually want to render the filtered expenses, it might not be necessary to also check if `expenses` is greater than 0. Like, expenses is always greater than 0 when filtered expenses is greater than 0, isn't it?

Reply:
sure! we are not using expenses anymore since I removed `the beginning of time`. thx!